### ISSUE TITLE
Documentation for the SPI-lines

### ISSUE THREAD

**Issue by arildlangseid** [fablab-bayreuth/coinspi#2]
Please update documentation for where to put the two signals for SPI:
find chip 74hc164:
SPI Data-signal is on pin1 and pin2. SPI Clock is on pin8.

Trouble is..... on my ch926 the signal on SPI is 9-bit.... which gives me problems using hardware SPI on Arduino :(

**Comment by Cat-Ion** [comment]
I'm not at the fablab anymore and haven't seen the hardware in years, so I won't be able to help much. It sounds rather unlikely that your CH-926 is that different - the 74HC164 can't do 9 bits, so you'd need either more of those or a different IC, but if that is indeed what's going on with yours then you might be able to use one of the ATTinys with an USI.

**Comment by arildlangseid** [comment]
You are correct - the 74HC164 is 8bit - so why the chineeze send 9 clock pulses is a mystery to me. I had problems using hardware SPI so I connected the Logic Analyzer which easely pinpointed why I had problems using hardware SPI... One short clock pulse before the real and useful 8 clock pulses came good in..... I have managed with an arduino I had laying around. But thanks for the tip about the ATTinys with USI. Will check it up.

**Comment by foreverchris** [comment]
how can i compile the code that i can download it ?
i only know arduino

**Comment by arildlangseid** [comment]
> how can i compile the code that i can download it ? i only know arduino

You might need Atmel studio for this project.

Not very well documented either - but you can have a look at my project.
https://github.com/arildlangseid/coinAcceptorInterface

Best regards, Arild
